docs/config/05-plugins.md

```diff
@@ -22,26 +23,35 @@ Therefore, a simple way to install a plugin is:
 npm install karma-<plugin name> --save-dev
 ```
 
-
 ## Loading Plugins
-By default, Karma loads all sibling NPM modules which have a name starting with `karma-*`.
 
-You can also explicitly list plugins you want to load via the `plugins` configuration setting. The configuration value can either be
-a string (module name), which will be required by Karma, or an object (inlined plugin).
+By default, Karma loads plugins from all sibling NPM packages which have a name starting with `karma-*`.
+
+You can also override this behavior and explicitly list plugins you want to load via the `plugins` configuration setting:
 
 ```javascript
-plugins: [
-  // Karma will require() these plugins
-  'karma-jasmine',
-  'karma-chrome-launcher'
-
-  // inlined plugins
-  {'framework:xyz': ['factory', factoryFn]},
-  require('./plugin-required-from-config')
-]
+config.set({
+  plugins: [
+    // Load a plugin you installed from NPM.
+    require('karma-jasmine'),
+
+    // Load a plugin from the file in your project.
+    require('./my-custom-plugin'),
+  
+    // Define a plugin inline.
+    { 'framework:xyz': ['factory', factoryFn] },
+
+    // Specify a module name or path which Karma will require() and load its 
+    // default export as a plugin.
+    'karma-chrome-launcher',
+    './my-fancy-plugin'
+  ]
+})
 ```
 
-There are already many [existing plugins]. Of course, you can write [your own plugins] too!
+## Activating Plugins
+
+Adding a plugin to the `plugins` array only makes Karma aware of the plugin, but it does not activate it. Depending on the plugin type you'll need to add a plugin name into `frameworks`, `reporters`, `preprocessors`, `middleware` or `browsers` configuration key to activate it. For the detailed information refer to the corresponding plugin documentation or check out [Developing plugins][developing plugins] guide for more in-depth explanation of how plugins work.
```

Comment:
This is a common source of errors, not doing all of the steps. Can we have a preview earlier, just before "installing" which calls out the three steps?  Then even better, each subsection of the three steps can call out:
`Installing plugins is the first of three steps in their use....`
`Loading plugins is the second of three steps in their use...`
`Activating plugins is the third and final step in their uses...`

Reply:
I agree with the point, but the current prose is not written as a step-by-step guide and I hesitate to rewrite the whole article to fix this comment. How about doing this in a separate PR later on?

Reply:
Perhaps my suggestion was not clear.  I'm just suggesting that we have 4 sentences that help users realize that one step is not enough.